diff -ruN1 a/build/tizen/CMakeLists.txt b/build/tizen/CMakeLists.txt
--- a/build/tizen/CMakeLists.txt	2020-08-19 10:32:58.552210200 +0100
+++ b/build/tizen/CMakeLists.txt	2020-08-19 14:37:43.760405500 +0100
@@ -113,6 +113,3 @@
   IF( USE_DEFAULT_RESOURCE_DIR )
-    SET( INSTALL_SHARE_DIR "${PREFIX}/share" )
-    IF( VCPKG_TOOLKIT_BUILD )
-      SET( INSTALL_SHARE_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/share" )
-    ENDIF()
+    SET( INSTALL_SHARE_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/share" )
 
@@ -319,6 +316,2 @@
 IF( INSTALL_CMAKE_MODULES )
-  IF( ENABLE_DEBUG )
-    SET( BIN_DIR "${BIN_DIR}/debug" )
-    SET( LIB_DIR "${LIB_DIR}/debug" )
-  ENDIF()
 
@@ -348,3 +341,3 @@
   IF( ENABLE_DEBUG )
-    install( FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/${name}.pdb DESTINATION ${BIN_DIR} )
+    install( FILES ${CMAKE_CURRENT_BINARY_DIR}/${name}.pdb DESTINATION ${BIN_DIR} )
   ENDIF()
